doc/content/syntax/zapdos_only.md

```diff
@@ -1,7 +1,12 @@
 # Zapdos-only Input File Syntax
 
 Listed below are the input parameter options for a Zapdos input file for code contained solely
+<<<<<<< HEAD
+within Zapdos (not within MOOSE or its modules). Click the blue link shown to the right of each
+heading to see a more detailed description of its purpose.
+=======
 within Zapdos (not within Squirrel, CRANE, or MOOSE and its modules). Click the blue link to see more detailed information about the usage of each object.
+>>>>>>> upstream/devel
```

Comment:
@csdechant I will assign this PR to you for review, but I wanted to point this out since I caught it while skimming the changes:

```suggestion
within Zapdos (not within Squirrel, CRANE, or MOOSE and its modules). Click the blue link to see more detailed information about the usage of each object.
```